package.json

```diff
@@ -76,7 +76,7 @@
     "jest": "^26.6.3",
     "jest-cli": "^26.6.3",
     "jsx-dom": "^6.4.21",
-    "kompendium": "^0.7.3",
+    "kompendium": "^0.8.5",
```

Comment:
`limel-date-picker` looks ok for datetime, date, time, and week, but for month, quarter, and year, the options look disabled:

![image](https://user-images.githubusercontent.com/154725/118778289-3a6aaf80-b88a-11eb-8480-a6856f6b9172.png)

Reply:
This still looks the same.

Reply:
seems to be fixed too

Reply:
This still looks the same as the screenshot above in my browser, but I don't think it has to stop us from merging.